Add builds for Linux aarch64 & ppc64le

martin-g opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

martin-g committed:
Add builds for Linux aarch64 & ppc64le

Signed-off-by: Martin Tzvetanov Grigorov <mgrigorov@apache.org>

martin-g commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13281691716) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/13281691716](https://github.com/conda-forge/conda-forge-webservices/actions/runs/13281691716). Examine the logs at this URL for more detail.</sub>

martin-g commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 25.1.1, conda-smithy 3.45.4, and conda-forge-pinning 2025.02.12.08.02.13

martin-g committed:
Do not build for linux-ppc64le

It segfaults while trying to load the library:
```
> library('phangorn')
2025-02-12T09:15:38.3852838Z Loading required package: ape
2025-02-12T09:15:47.0076067Z code for methods in class “Rcpp_Fitch” was not checked for suspicious field assignments (recommended package ‘codetools’ not available?)
2025-02-12T09:15:47.2981478Z code for methods in class “Rcpp_Fitch” was not checked for suspicious field assignments (recommended package ‘codetools’ not available?)
2025-02-12T09:15:47.5403786Z >
2025-02-12T09:15:47.5404730Z >
2025-02-12T09:16:01.0393833Z /home/conda/feedstock_root/build_artifacts/r-phangorn_1739351512097/test_tmp/run_test.sh: line 7:  1124 Segmentation fault      (core dumped) $R -e "library('phangorn')"
2025-02-12T09:16:01.5048775Z WARNING: Tests failed for r-phangorn-2.12.1-r44h63c5bbe_1.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
2025-02-12T09:16:01.5485693Z Traceback (most recent call last):
2025-02-12T09:16:01.5486399Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/build.py", line 3484, in test
2025-02-12T09:16:01.5501050Z     utils.check_call_env(
2025-02-12T09:16:01.5501600Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/utils.py", line 406, in check_call_env
2025-02-12T09:16:01.5509938Z     return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
2025-02-12T09:16:01.5516278Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-02-12T09:16:01.5516756Z   File "/opt/conda/lib/python3.12/site-packages/conda_build/utils.py", line 382, in _func_defaulting_env_to_os_environ
2025-02-12T09:16:01.5517305Z     raise subprocess.CalledProcessError(proc.returncode, _args)
2025-02-12T09:16:01.5517751Z subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/home/conda/feedstock_root/build_artifacts/r-phangorn_1739351512097/test_tmp/conda_test_runner.sh']' returned non-zero exit status 139.
2025-02-12T09:16:01.5518032Z
```

Signed-off-by: Martin Tzvetanov Grigorov <mgrigorov@apache.org>

martin-g commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 25.1.1, conda-smithy 3.45.4, and conda-forge-pinning 2025.02.12.08.02.13

martin-g commented:
@conda-forge-admin, please ping team
